current directory: /home/vopuu/project/lbe_website/vendor/bundle/ruby/3.0.0/gems/ffi-1.9.25/ext/ffi_c
/usr/bin/ruby3.0 -I /usr/lib/ruby/vendor_ruby -r ./siteconf20240907-690313-17f9m9.rb extconf.rb
checking for ffi_call() in -lffi... yes
checking for ffi_closure_alloc()... yes
checking for ffi_raw_call()... yes
checking for ffi_prep_raw_closure()... yes
checking for shlwapi.h... no
checking for ruby/thread.h... yes
checking for rb_thread_blocking_region()... no
checking for rb_thread_call_with_gvl()... yes
checking for rb_thread_call_without_gvl()... yes
checking for ffi_prep_cif_var()... yes
creating extconf.h
creating Makefile

current directory: /home/vopuu/project/lbe_website/vendor/bundle/ruby/3.0.0/gems/ffi-1.9.25/ext/ffi_c
make DESTDIR\= clean

current directory: /home/vopuu/project/lbe_website/vendor/bundle/ruby/3.0.0/gems/ffi-1.9.25/ext/ffi_c
make DESTDIR\=
compiling AbstractMemory.c
AbstractMemory.c: In function ‘memory_put_bytes’:
AbstractMemory.c:586:9: warning: implicit declaration of function ‘rb_safe_level’ [-Wimplicit-function-declaration]
  586 |     if (rb_safe_level() >= 1 && OBJ_TAINTED(str)) {
      |         ^~~~~~~~~~~~~
compiling ArrayType.c
compiling Buffer.c
compiling Call.c
Call.c: In function ‘rbffi_SetupCallParams’:
Call.c:303:25: warning: implicit declaration of function ‘rb_safe_level’ [-Wimplicit-function-declaration]
  303 |                     if (rb_safe_level() >= 1 && OBJ_TAINTED(argv[argidx])) {
      |                         ^~~~~~~~~~~~~
Call.c: In function ‘rbffi_CallFunction’:
Call.c:412:20: warning: passing argument 1 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  412 |         rb_rescue2(rbffi_do_blocking_call, (VALUE) bc, rbffi_save_frame_exception, (VALUE) &frame, rb_eException, (VALUE) 0);
      |                    ^~~~~~~~~~~~~~~~~~~~~~
      |                    |
      |                    VALUE (*)(void *) {aka long unsigned int (*)(void *)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Call.c:45:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:18: note: expected ‘VALUE (*)(VALUE)’ {aka ‘long unsigned int (*)(long unsigned int)’} but argument is of type ‘VALUE (*)(void *)’ {aka ‘long unsigned int (*)(void *)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                  ^~~~~~~~~~~~~~~
Call.c:412:56: warning: passing argument 3 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  412 |         rb_rescue2(rbffi_do_blocking_call, (VALUE) bc, rbffi_save_frame_exception, (VALUE) &frame, rb_eException, (VALUE) 0);
      |                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                        |
      |                                                        VALUE (*)(void *, VALUE) {aka long unsigned int (*)(void *, long unsigned int)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Call.c:45:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:40: note: expected ‘VALUE (*)(VALUE,  VALUE)’ {aka ‘long unsigned int (*)(long unsigned int,  long unsigned int)’} but argument is of type ‘VALUE (*)(void *, VALUE)’ {aka ‘long unsigned int (*)(void *, long unsigned int)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                                        ^~~~~~~~~~~~~~~~~~~~~
compiling ClosurePool.c
compiling DataConverter.c
compiling DynamicLibrary.c
compiling Function.c
Function.c: In function ‘callback_with_gvl’:
Function.c:740:16: warning: passing argument 1 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  740 |     rb_rescue2(invoke_callback, (VALUE) data, save_callback_exception, (VALUE) data, rb_eException, (VALUE) 0);
      |                ^~~~~~~~~~~~~~~
      |                |
      |                VALUE (*)(void *) {aka long unsigned int (*)(void *)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Function.c:49:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:18: note: expected ‘VALUE (*)(VALUE)’ {aka ‘long unsigned int (*)(long unsigned int)’} but argument is of type ‘VALUE (*)(void *)’ {aka ‘long unsigned int (*)(void *)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                  ^~~~~~~~~~~~~~~
Function.c:740:47: warning: passing argument 3 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  740 |     rb_rescue2(invoke_callback, (VALUE) data, save_callback_exception, (VALUE) data, rb_eException, (VALUE) 0);
      |                                               ^~~~~~~~~~~~~~~~~~~~~~~
      |                                               |
      |                                               VALUE (*)(void *, VALUE) {aka long unsigned int (*)(void *, long unsigned int)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Function.c:49:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:40: note: expected ‘VALUE (*)(VALUE,  VALUE)’ {aka ‘long unsigned int (*)(long unsigned int,  long unsigned int)’} but argument is of type ‘VALUE (*)(void *, VALUE)’ {aka ‘long unsigned int (*)(void *, long unsigned int)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                                        ^~~~~~~~~~~~~~~~~~~~~
Function.c: In function ‘callback_prep’:
Function.c:951:5: warning: ‘ffi_prep_closure’ is deprecated: use ffi_prep_closure_loc instead [-Wdeprecated-declarations]
  951 |     ffiStatus = ffi_prep_closure(code, &fnInfo->ffi_cif, callback_invoke, closure);
      |     ^~~~~~~~~
In file included from Function.c:54:
/usr/include/x86_64-linux-gnu/ffi.h:347:1: note: declared here
  347 | ffi_prep_closure (ffi_closure*,
      | ^~~~~~~~~~~~~~~~
compiling FunctionInfo.c
compiling LastError.c
compiling LongDouble.c
LongDouble.c: In function ‘rbffi_longdouble_new’:
LongDouble.c:19:59: warning: passing argument 3 of ‘rb_rescue’ from incompatible pointer type [-Wincompatible-pointer-types]
   19 |         rb_cBigDecimal = rb_rescue(bigdecimal_load, Qnil, bigdecimal_failed, rb_cObject);
      |                                                           ^~~~~~~~~~~~~~~~~
      |                                                           |
      |                                                           VALUE (*)(VALUE) {aka long unsigned int (*)(long unsigned int)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from LongDouble.h:33,
                 from LongDouble.c:1:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:50:39: note: expected ‘VALUE (*)(VALUE,  VALUE)’ {aka ‘long unsigned int (*)(long unsigned int,  long unsigned int)’} but argument is of type ‘VALUE (*)(VALUE)’ {aka ‘long unsigned int (*)(long unsigned int)’}
   50 | VALUE rb_rescue(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE);
      |                                       ^~~~~~~~~~~~~~~~~~~~~
compiling MappedType.c
compiling MemoryPointer.c
compiling MethodHandle.c
compiling Platform.c
compiling Pointer.c
compiling Struct.c
compiling StructByReference.c
compiling StructByValue.c
compiling StructLayout.c
compiling Thread.c
compiling Type.c
compiling Types.c
compiling Variadic.c
Variadic.c: In function ‘variadic_invoke’:
Variadic.c:274:20: warning: passing argument 1 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  274 |         rb_rescue2(rbffi_do_blocking_call, (VALUE) bc, rbffi_save_frame_exception, (VALUE) &frame, rb_eException, (VALUE) 0);
      |                    ^~~~~~~~~~~~~~~~~~~~~~
      |                    |
      |                    VALUE (*)(void *) {aka long unsigned int (*)(void *)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Variadic.c:43:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:18: note: expected ‘VALUE (*)(VALUE)’ {aka ‘long unsigned int (*)(long unsigned int)’} but argument is of type ‘VALUE (*)(void *)’ {aka ‘long unsigned int (*)(void *)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                  ^~~~~~~~~~~~~~~
Variadic.c:274:56: warning: passing argument 3 of ‘rb_rescue2’ from incompatible pointer type [-Wincompatible-pointer-types]
  274 |         rb_rescue2(rbffi_do_blocking_call, (VALUE) bc, rbffi_save_frame_exception, (VALUE) &frame, rb_eException, (VALUE) 0);
      |                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                        |
      |                                                        VALUE (*)(void *, VALUE) {aka long unsigned int (*)(void *, long unsigned int)}
In file included from /usr/include/ruby-3.0.0/ruby/ruby.h:38,
                 from /usr/include/ruby-3.0.0/ruby.h:38,
                 from Variadic.c:43:
/usr/include/ruby-3.0.0/ruby/internal/iterator.h:51:40: note: expected ‘VALUE (*)(VALUE,  VALUE)’ {aka ‘long unsigned int (*)(long unsigned int,  long unsigned int)’} but argument is of type ‘VALUE (*)(void *, VALUE)’ {aka ‘long unsigned int (*)(void *, long unsigned int)’}
   51 | VALUE rb_rescue2(VALUE(*)(VALUE),VALUE,VALUE(*)(VALUE,VALUE),VALUE,...);
      |                                        ^~~~~~~~~~~~~~~~~~~~~
compiling ffi.c
linking shared-object ffi_c.so

current directory: /home/vopuu/project/lbe_website/vendor/bundle/ruby/3.0.0/gems/ffi-1.9.25/ext/ffi_c
make DESTDIR\= install
/usr/bin/install -c -m 0755 ffi_c.so ./.gem.20240907-690313-3uqcrc
